ISSUE_TEMPLATE/BUG_TEMPLATE.md

```diff
@@ -2,7 +2,7 @@
 name: 🐛 Bug report
 about: Create a report to help us improve
 title: "[BUG]"
-labels: bug
+labels: 'bug, untriaged, Beta'
```

Comment:
It looks like all other labels are lowercase. Is there a reason why Beta is not?

Reply:
There is no particular reason for it.